src/amo/components/Header/index.js

```diff
@@ -60,12 +83,35 @@ export class HeaderBase extends React.Component {
           <DownloadFirefoxButton
             className="Header-download-button Header-button"
           />
-          <AuthenticateButton
-            className="Header-authenticate-button Header-button Button--action
+
+          {this.props.isAuthenticated ? (
+            <DropdownMenu
```

Comment:
I didn't know where to put this but this shouldn't have a cursor that changes:

![2017-09-11 11 18 10](https://user-images.githubusercontent.com/90871/30270007-f2560fa4-96e2-11e7-84e8-724c3bbabb23.gif)

It's confusing because clicking on my username doesn't do anything. So I don't think that button, when signed in, should have a cursor style. That might mean an extra `className` when the user is signed in so we can style it differently.